### DISCUSSION TITLE
Support for Creabest BMS

### DISCUSSION THREAD

**Discussion by addigaz** [fl4p/batmon-ha#183] [Ideas]
Hi,
if possible please add support for Creabest BMS. 
Please let me know if this is possible and if I can help in any way.
Thanks in advance

**Comment by Alex-DeIt** [comment]
Would be very nice.
I am getting strange sinusoidal values related to all monitored parameters.
![image](https://github.com/fl4p/batmon-ha/assets/83119676/3764951d-af3e-4dad-b7bb-30d73db93d37)
